### DISCUSSION TITLE
Frage zum Shellskript (excercise04.sh)

### DISCUSSION THREAD

**Discussion by Foxglovve** [irgroup-classrooms/dis08-2021#44] [Q&A]
In der Aufgabe steht, wir sollen pro Lösung einen commit durchführen. Heißt das, ich lege meine Datei excersise04.sh nach und nach an, so dass immer ein Befehl dazu kommt, die Datei geadded und commited wird und dann der nächste Befehl für die nächste Lösung dazu kommt? Das habe ich versucht, aber die Datei lässt sich nicht mehr bearbeiten, ist ja ein Skript und keine Textdatei.

Wenn aber alle Befehle von Anfang an in der Datei stehen, sehe ich den Sinn von 6 einzelnen Commits nicht? Nach dem ersten Commit ändert sich doch dann nichts mehr an der Datei.
Können Sie mir das irgendwie näher bringen?

**Comment by Foxglovve** [comment]
Oder legen wir die nach und nach im Repository an und commiten dann jeweils? Ich hatte es ursprünglich im Texteditor angelegt und dann auf .sh enden lassen in meinem local Repository.

**Comment by HaakFab** [comment]
Hallo,
"We want to see one commit in GitHub per solution!"
Soll heissen, nicht für jede Teilaufgabe, sonder NUR ein Commit für die ganze Aufgabe. 
Aber wie immer in Git Fragen: So lange am ende die Datei richtig benannt mit korrektem Inhalt an der richtigen Stelle ist, ist uns nicht wichtig in wie vielen Commits und ob über die Kommandozeile commitet oder im Browser hochgeladen wurde.
Also mein Tip für die Aufgabe: Machen Sie das bei sich "offline" fertig, erstellen Sie Ihr Bash Script und Committen Sie dann oder laden Sie über das Web Interface hoch.
VG,
Fabian Haak

**Comment by Foxglovve** [comment]
Ok, dankeschön, ich habs jetzt für mich gelöst.
Was den richtigen Platz angeht:
Mein Ursprungsrepository ist leider komplett hinüber. Ich habe deshalb ein neues Repository das ganz klassisch "new" heißt angelegt und dort nochmal alle Aufgaben abgelegt. Klappt jetzt auch deutlich besser. Aber können Sie das bitte berücksichtigen? Bzw. können Sie das überhaupt sehen? https://github.com/Foxglovve/new
